src/nimibook/docs.nim

```diff
@@ -7,11 +7,18 @@ proc useNimibook*(nbDoc: var NbDoc) =
   let
     nbThisFile = changeFileExt(nbDoc.filename.AbsoluteFile, ".nim")
     thisTuple = nbThisFile.splitFile
-    nbThisDir: AbsoluteDir = thisTuple.dir
-    nbHomeDir: AbsoluteDir = findNimbleDir(nbThisDir) / "docs".RelativeDir
-    nbSrcDir = nbHomeDir / RelativeDir(".." / "book")
-
-  # are these two actually needed? well, home_path is needed in path_to_root, but other than that?
+    pathToRootFolder = getEnv("nimibook_path_to_rootfolder")
+    nbThisDir: AbsoluteDir = pathToRootFolder.toAbsoluteDir
+    nbHomeDir: AbsoluteDir = nbThisDir / RelativeDir("..") / "docs".RelativeDir
```

Comment:
`nbHomeDir` is never used, could that be the problem?

Reply:
[This](https://github.com/pietroppeter/nimib/blob/dda83a58e1fb2ae53523e29905614f9bfa62a02a/src/nimib.nim#L77) line uses the `nbHomeDir` injected by `nbInit` to decide where to write the file.

Reply:
Good catch. So it's something to be fixed in Nimib.